### PR TITLE
Lizard skin handbag

### PR DESCRIPTION
**Pull request by ZomgPonies**
## What Does This PR Do
Adds a handbag/purse made out of lizard skin, which is gotten from gibbing unathi. Human skin had a crafting recipe to make a human skinsuit, I thought this would be a fun addition to put the lizard skin to use outside of using it as a rug. Requires 5 skins to make a satchel.

## Why It's Good For The Game
The lizard skin was already in-game and had a fun cartoony look, I thought it was a shame it didn't actually have any utility. Follows in the same vein as the 'kentucky fried vox' PR.

## Images of changes
original lizard skin sprite:
lizard-skin
handbag/purse sprite:
skin bag
worn sprite:
Capture

## Changelog
:cl:
add: added lizard skin satchel and recipe
imageadd: sprites for lizard skin satchel by Travelling Merchant
/:cl:

### CODE REVIEW

File: code/game/objects/items/stacks/sheets/leather.dm [hunk 1]
```diff
@@ -48,6 +48,14 @@ GLOBAL_LIST_INIT(human_recipes, list( \
 	singular_name = "lizard skin piece"
 	icon_state = "sheet-lizard"
 
+GLOBAL_LIST_INIT(lizard_recipes, list( \
+	new/datum/stack_recipe("lizard skin handbag", /obj/item/storage/backpack/satchel/lizard, 5, on_floor = 1), \
```
Comment (farie82) [inline code/game/objects/items/stacks/sheets/leather.dm:52]: `TRUE` here

**Comment by MRgaty** [comment]
This is hilarious

**Comment by nok-ko** [comment]
Fantastic work.

**Comment by Muhdi345** [comment]
We really need a new traitor objective for these kinds of items. I'd love to make a handbag of the local unathi's.

**Comment by ZomgPonies** [comment]
> 
> 
> We really need a new traitor objective for these kinds of items. I'd love to make a handbag of the local unathi's.

That's a bad idea, since gibbing active people effectively removes them completely from the round. So it's best to leave it as is, and people could technically just go in the morgue and instead gib uncloneables.

**Pull request merged**